core/domain/acl_decorators.py

```diff
@@ -68,6 +76,39 @@ def check_activity_accessible(
             activity_rights.viewable_if_private)
 
 
+def check_exploration_editable(user_id, user_actions, exploration_rights):
+    """Returns a boolean to signify whether given exploration is editable
+    by the user or not.
+
+    Args:
+        user_id: str. Id of the given user.
+        user_actions: list(str). List of actions given user can perform.
+        exploration_rights: Exploration rights object.
+
+    Returns:
+        bool. Whether the given exploration can be accessed.
+    """
+    if exploration_rights.community_owned:
+        return True
```

Comment:
Even if user id is None?

(If None is allowed as a possible arg make sure to specify that in the docstring.)